src/request/sign-transaction/SignTransactionApi.js

```diff
@@ -83,9 +83,9 @@ class SignTransactionApi extends TopLevelApi {
     }
 }
 
-/** @type {{[layout: string]: any, standard: typeof LayoutStandard, checkout: typeof LayoutStandard}} */
+/** @type {{[layout: string]: string}} */
 SignTransactionApi.Layouts = {
-    standard: LayoutStandard,
-    checkout: LayoutStandard,
-    // cashlink: LayoutCashlink,
+    standard: 'standard',
+    checkout: 'checkout',
+    cashlink: 'cashlink',
```

Comment:
The variables should be uppercase: `STANDARD`, `CHECKOUT`, `CASHLINK`, if possible.